src/parse-result.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Parse result-specific refract elements.
+ *
+ * General structure:
+ *
+ * + ParseResult
+ *   + Annotation
+ */
+
+import apiDescription from 'minim-api-description';
+
+export function namespace(options) {
+  const minim = options.base;
+  const ArrayElement = minim.getElementClass('array');
+  const StringElement = minim.getElementClass('string');
+
+  class ParseResult extends ArrayElement {
+    constructor() {
+      super(...arguments);
+      this.element = 'parseResult';
+    }
+
+    get api() {
+      return this.children(item => item.classes.contains('api')).first();
```

Comment:
Shouldn't this be wrapped using API description namespace? Also please test an api description namespace helper on this to see if this is getting correctly wrapped or not.